composer.patches.json

```diff
@@ -0,0 +1,20 @@
+{
+    "patches": {
```

Comment:
I'd prefer to have these in the main composer.json not to have another new file in the docroot. but that's probably personal preference.

Reply:
We're going to make a trade ... :rofl: 

I'll remove `composer.patches.json`, but revert directory do `patches`.

Reason: symplify/vendor-patches

- it adds generated patches to `composer.json`, not `composer.patches.json`.
- it creates patch file in `patches`, not `.patches`

Reply:
mmmm ok, but we should add this new files/directory to some .htaccess with "deny from all", to avoid possibly leaking the openmage version thru the existance of these files

Reply:
Added .htaccess.